docs/modules/ROOT/pages/release-notes/release-notes-24-0.adoc

```diff
@@ -0,0 +1,39 @@
+= Payara Enterprise 5.24.0 Release Notes
+
+== Supported APIs and Applications
+
+* Java EE 8 Applications
+* MicroProfile 3.3
+* Jakarta EE 8 Applications
+
+== New Features
+
+* [FISH-656] - CLI Upgrade Tool MVP for Payara Server Enterprise
+
+== Improvements
+
+* [FISH-152] - Mark applications as "unavailable" as soon as they are disabled
```

Comment:
Many of these improvements are released in 5.23.1, so they should not be in this release notes.

Reply:
My bad! Got the copy paste mixed up somehow, I'll fix that now